satzung.md

```diff
@@ -63,60 +63,71 @@ Die Organe der Fachschaft sind:
 
  
-## §5d. Beschlüsse
+## §4f Beschlüsse
 
 (1) Der Fachschaftsrat ist beschlussfähig, wenn mindestens die Hälfte seiner Mitglieder anwesend ist, wobei mindestens zwei Mitglieder persönlich anwesend sein müssen.
 
-(2) Die Beschlüsse werden mit der einfachen Mehrheit aller gültigen von anwesenden Mitgliedern abgegebenen Stimmen gefasst.
+(2) Die Beschlüsse werden mit der einfachen Mehrheit aller gültigen von anwesenden Mitgliedern abgegebenen Stimmen gefasst. Bei Stimmengleichheit entscheidet die Stimme des Sitzungsleitenden.
 
 (3) Die Beschlüsse des Fachschaftsrates werden in Protokollen festgehalten.
 
 
-## §5e. Sitzungen
+## §4g Sitzungen
 
 (1) Der Fachschaftsrat soll während der Vorlesungszeit jede Woche zusammentreten. Der Termin der turnusmäßigen Sitzungen ist durch eine E-Mail an die Fachschaft bekanntzugeben.
 
 (2) Zu außerordentlichen Sitzungen muss eine Einladung per E-Mail an die Fachschaft erfolgen.
 
-(3) Zu Beginn jeder Sitzung ist vom Vorsitzenden ein Protokollführer festzulegen, der alle Beschlüsse der Sitzung und diskutierte Punkte in einem Protokoll festhält. Der Protokollführer muss persönlich anwesend sein. Weiteres regelt §5f.
+(3) Zu Beginn jeder Sitzung ist vom Sitzungsleitenden ein Protokollführender festzulegen, der alle Beschlüsse der Sitzung und diskutierte Punkte in einem Protokoll festhält. Der Protokollführende muss persönlich anwesend sein. Weiteres regelt §4h.
 
-(4) Die Sitzungen des Fachschaftsrates sind offen für alle Mitglieder der Fachschaft. Auf Beschluss kann die Öffentlichkeit ausgeschlossen werden. Inhalte von nicht öffentlichen Sitzungen sind in gesonderten Protokollen festzuhalten, auf die mit einer Begründung für die Geheimhaltung im regulären Protokoll verwiesen wird.
+(4) Die Sitzungen des Fachschaftsrates sind offen für alle Mitglieder der Fachschaft. Auf Beschluss kann die Öffentlichkeit ausgeschlossen werden. Inhalte von nicht öffentlichen Sitzungsteilen sind in gesonderten Protokollen festzuhalten, auf die mit einer Begründung für die Geheimhaltung im regulären Protokoll verwiesen wird.
 
 
-## §5f. Sitzungsprotokolle
+## §4h Sitzungsprotokolle
 
 (1) Das Protokoll jeder Sitzung muss mindestens die folgenden Punkte enthalten:
 
 1. Datum
 2. Anwesende Mitglieder
-3. Protokollführer
+3. Protokollführender
 4. Beschreibung der Anträge und Beschlüsse mit genauen Abstimmungsergebnissen
 
-(2) Die Sitzungsprotokolle werden spätestens drei Vorlesungstage nach der Sitzung veröffentlicht.
+(2) Die Sitzungsprotokolle werden allen FSR-Mitgliedern unverzüglich nach der Sitzung zugänglich gemacht.
+
+(3) Wenn innerhalb von zwei Werktagen nach dem Tag der Sitzung keine Änderungswünsche beim Protokollführenden eingegangen sind, gilt das Protokoll als angenommen. Dies gilt auch wenn während der Sitzung keine Beschlussfähigkeit bestand.
+
+(4) Angenommene Protokolle werden nach spätestens drei Vorlesungstagen durch den Sitzungsleitenden veröffentlicht.
```

Comment:
Wollen wir hier nicht die Lücke in der Vorlesungsfreien Zeit schließen? Ich wäre für 5 Tage oder so